pkgs/applications/audio/uhhyou.lv2/default.nix

```diff
@@ -0,0 +1,75 @@
+{ stdenv, fetchgit, pkgconfig, libjack2, liblo, libGL, libX11, lv2, fftw }:
+
+stdenv.mkDerivation rec {
+  pname = "uhhyou.lv2";
+  version = "unstable-2020-05-27";
+
+  src = fetchgit {
+    url = "https://github.com/ryukau/LV2Plugins";
+    rev = "f20d1c8a624d3a757f820b5c41c70f6f9bb6dc22";
+    deepClone = true;
+    sha256 = "12g9j4rrhkw91sbnqi8xg8wkl5j34m0l7fdwhqa2ny526n52lq7k";
+  };
+
+  buildInputs = [ libjack2 liblo libGL libX11 lv2 fftw ];
+
+  nativeBuildInputs = [ pkgconfig ];
+
+  # Temporary patch to DPF. See: https://github.com/DISTRHO/DPF/issues/216
+  prePatch = ''
+    # patchShebangs dpf/utils/generate-ttl.sh
+    patchShebangs generate-ttl.sh
+    cp patch/NanoVG.cpp lib/DPF/dgl/src/NanoVG.cpp
+  '';
+
+  enableParallelBuilding = true;
+
+  installPhase = ''
```

Comment:
Why not remove it and let `make install` do it all? Upstream seems to be very instructive: https://github.com/ryukau/LV2Plugins#building

Reply:
``make install`` only  installs the lv2's and the config, but it istalls to ~/, hardcoded, and it uses the makefile and a python script. 
So this was the easier and cleaner solution.

Reply:
Ideally, you should request upstream to provide a system wide, and [FHS](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard) compatible Makefile. What could be even better is doing it for them in a PR, and then apply that PR as a patch. Could you please do either of the two and link a comment explaining exactly this:

> make install only installs the lv2's and the config, but it istalls to ~/, hardcoded, and it uses the makefile and a python script.
So this was the easier and cleaner solution.

Plus a link to an issue upstream?

Reply:
Also, this review was not addressed.